### PR TITLE
[DOCS-223] Best Practices: Protect against SSRF

### PR DESCRIPTION
**Pull request by ana-dashuk-cobalt**
Hi @harsh-bothra

Thank you for your pull request, https://github.com/cobalthq/cobalt-product-public-docs/pull/150

I've tried a rewrite of your content, and set it up in this PR. When I rewrote your content, I tried to:

- Keep your messaging
- Improve readability
- Focus on [Active Voice](https://developers.google.com/style/voice)

Please let me know if there are any mistakes. If you can verify the programming language that I applied to your code samples, that would be great. I'm not 100% sure about the language used.

This is how the PR will appear in our Product Docs: https://deploy-preview-155--cobalt-docs.netlify.app/bestpractices/defend-against-ssrf/

(Note: I can't make you a "Reviewer" as we have limits on our GitHub subscription. However, you should still be able to review each line of the PR.)

### CODE REVIEW

File: content/en/BestPractices/defend-against-ssrf.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
+---
+title: Protect against Server-Side Request Forgery
```
Comment (mjang-cobalt) [inline content/en/BestPractices/defend-against-ssrf.md:2]:
```suggestion
title: Protect Against Server-Side Request Forgery
```

File: content/en/BestPractices/defend-against-ssrf.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+linkTitle: Protect against SSRF
```
Comment (mjang-cobalt) [inline content/en/BestPractices/defend-against-ssrf.md:3]:
```suggestion
title: Protect Against Server-Side Request Forgery
linkTitle: Protect Against SSRF
```

File: content/en/BestPractices/defend-against-ssrf.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+
+## Attack Scenarios
+
+Let's see some examples of SSRF attacks.
+
+### Accessing a Localhost
+
+In the following code example, the application accepts user-supplied input through the `url` parameter and passes it to the `urllib.request.urlopen(url).read()` function. The application processes URLs without filtration, which may lead to an SSRF attack.
+
+An attacker may pass a URL such as `https://www.target.com/test/?url=http://127.0.0.1` to the vulnerable code and access the localhost `127.0.0.1`.
```
Comment (mjang-cobalt) [inline content/en/BestPractices/defend-against-ssrf.md:47]: Let's avoid "Naming names". Per RFC 2606, we can use generic names like example.com for documentation. So I suggest:

```suggestion
An attacker may pass a URL such as `https://www.example.com/test/?url=http://127.0.0.1` to the vulnerable code and access the localhost `127.0.0.1`.
```

Comment (mjang-cobalt) [reply content/en/BestPractices/defend-against-ssrf.md:47]: I'm going to _assume_ this change is OK, as we'd like to plan to "publish". @harsh-bothra, let us know if you see a problem.

**Review by mjang-cobalt — APPROVED** [review]
Approved unless there are additional changes, or objections to the changes that I've pushed to this PR.

After publication, we may edit this further. We're just starting this process, and may implement changes for the "next time".

**Comment by ana-dashuk-cobalt** [comment]
Hi @harsh-bothra,

You might have seen that we moved your article here after reviewing.

Please let us know if you have any comments. If you can verify that there are no major errors in your messaging, that would be great.

**Pull request merged**